index/server/pkg/server/endpoint.go

```diff
@@ -50,6 +50,8 @@ func serveDevfileIndex(c *gin.Context) {
 
 // serveDevfileIndexWithType returns the index file content with specific devfile type
 func serveDevfileIndexWithType(c *gin.Context) {
+	// Set headers
+	c.Header("Access-Control-Allow-Origin", "*")
```

Comment:
Let's discuss on today's call if there's any other endpoints we'll want to add this for. Minimally I'm thinking the `/index` endpoints (so `serveDevfileIndex` as well).

Reply:
Sounds good.